data/products/rancher-setup/packages.yaml

```diff
@@ -0,0 +1,36 @@
+packages:
+  _namespace_jeos:
+    package:
+      - patterns-base-minimal_base
+      - gio-branding-SLE
+  _namespace_sle_common:
```

Comment:
I elected to define this for the product, instead of using the common definition, because so many package in sle_common were not required (all of YaST, for example).